src/main/scala/org/ergoplatform/local/TransactionGenerator.scala

```diff
@@ -43,6 +45,7 @@ class TransactionGenerator(viewHolder: ActorRef,
 
       viewHolder ! GetDataFromCurrentView[ErgoHistory, UtxoState, ErgoWallet, ErgoMemPool, Unit] { v =>
         currentFullHeight = v.history.headersHeight
+        v.vault.publicKeys(0, 100).onComplete(_.foreach(pks => propositions = pks))
 
         context.system.eventStream.subscribe(self, classOf[SuccessfulTransaction[ErgoTransaction]])
```

Comment:
did you want this to be inside `GetDataFromCurrentView` handling by `NodeViewHolder`?